src/mlio/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_library(mlio
     memory/memory_block.cxx
     memory/memory_slice.cxx
     memory/util.cxx
+    record_readers/blob_record_reader.cxx
```

Comment:
Could you please move this line right above `record_readers/csv_record_reader.cxx`. In this list the files are sorted by directory and then by the file name.